ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh

```diff
@@ -15,11 +15,11 @@ ES_PASSWORD=$(cat "/secret/password")
 ES_USERNAME=$(cat "/secret/username")
 
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
-pushd e2e-benchmarking/workloads/kube-burner
-export WORKLOAD=node-density-heavy
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+export NAMESPACED_ITERATIONS=${NAMESPACED_ITERATIONS:-"true"}    
```

Comment:
@svetsa-rh I the kube-burner version of node-density-heavy, all objects are created in one namespace:
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/725a0890c770844347594aa8f6fa0791be7c98b6/workloads/kube-burner/README.md?plain=1#L102
Why do you want to change NAMESPACED_ITERATIONS to true here?
One problem with NAMESPACED_ITERATIONS as true is the test running time will be longer. As in one namespace, resources are created with some parallel depending on QPS and BURST, but when NAMESPACED_ITERATIONS is true, objects will be created iteration by iteration. In your current config PODS_PER_NODE: "100" with 3 workers, that will be 300 iterations.

Reply:
@qiliRedHat Couple of things here, I am only setting PODS_PER_NODE: "100" for us to verify that the job works and passes the test. We have seen latency issues with node-density tests. See https://issues.redhat.com/browse/OCPQE-15525.
I am setting this to a number where we can verify that the test is working overall and passes so the PR can be merged. Later, we should do experiments and update the value similar to what Dev has and close to defaults. 

As I understand, node-density-heavy with NAMESPACED_ITERATIONS=true improves performance and stability.  This is the new default starting kube-burner v1.7.1.

Reply:
Based on this pr I don't think the namespaceIterations is in yet: https://github.com/cloud-bulldozer/kube-burner/pull/362/files

Reply:
That's correct, Paige. node-density-heavy is happier with the changes merged in https://github.com/cloud-bulldozer/kube-burner/pull/349 . When there is a release with those defaults, at that point it makes sense to update e2e-benchmarking with the kube-burner release and migrate node-density-heavy to use ocp-wrapper workload.

Until then, classic kube-burner and NAMESPACED_ITERATIONS, as Sharada mentioned above.

Reply:
Thanks for the info! NAMESPACED_ITERATIONS=true together with iterationsPerNamespace: 1000 makes sense, that won't make the test taking too long time.